### PR TITLE
Splitting unit tests to be executed

### PR DESCRIPTION
**Pull request by gabriel-ga-nascimento**
Adding a mechanism to group tests so they can be spliced and sorted to run in parallel instances

### CODE REVIEW

File: documentation/commands/cmd-autotest.txt [hunk 1]
```diff
@@ -46,4 +46,6 @@
 ~ --selenium: 
 ~ Run the selenium tests.
 ~ --timeout=xxx: 
-~ Specify the timeout for the webclient (the value of the timeout in milliseconds, 0 for an infinite wait)
\ No newline at end of file
+~ Specify the timeout for the webclient (the value of the timeout in milliseconds, 0 for an infinite wait).
+~ --group=x:
+~ The test group to be executed. The four available options are 1, 2, 3 and 4.
```
Comment (felipealmeida-spec) [inline documentation/commands/cmd-autotest.txt:51]: Is this mandatory? Should we still have an empty that runs all tests?

Comment (gabriel-ga-nascimento) [reply documentation/commands/cmd-autotest.txt:51]: It's good to have it in the first stage. We can think about changing it after all of our tests are annotated with their corresponding groups

**Review by miklos-bl — APPROVED** [review]
LGTM 👍

**Review by felipealmeida-spec — APPROVED** [review]
Nice one!

**Pull request merged**